### DISCUSSION TITLE
v4.2.0

### DISCUSSION THREAD

**Discussion by MiguelRipoll23** [MiguelRipoll23/homebridge-securitysystem#107] [General]
This update improves the behavior of the new `Arming Lock` switch to block arming the security system and a new option to enable/disable the trigger delay during night mode and improved documentation for the server.

## Changelog
- Improved behavior and logging of the `Arming lock` switch, will affect all modes unless a condition is added to the automation. The internal counter has been removed to prevent out-of-sync issues.
- Added option `Trigger During Night Mode With Delay`.
- Added Wiki redirect to the `Server` page.

<hr /><em>This discussion was created from the release <a href='https://github.com/MiguelRipoll23/homebridge-securitysystem/releases/tag/v4.2.0'>v4.2.0</a>.</em>

**Comment by chrispan0** [comment]
Is it possible to remove the arming lock switch?

**Reply by MiguelRipoll23** [reply]
I just realized that it's being shown as default, this an error. Go to the switches section from sections and disable the option "Show Arming Lock Switch (beta)". Thanks!
